Revise Core note and add IMPORTANT format (#1896)
sensu/sensu-docs: content/sensu-core/1.8/_index.md

**These resources can help you migrate to [Sensu Go][24]**, the latest version of Sensu:

- [**Migration guide**][22]: Review this step-by-step guide to installing Sensu Go and using the [translator][23] command-line tool to transfer your Sensu Core checks, handlers, and mutators to Sensu Go.
- [**Sensu Go documentation**][24]: Learn about the Sensu Go architecture and APIs. The [Sensu Go migration section][25] includes an overview of Sensu Go features.
- [**Sensu Community Slack**][26]: Join hundreds of other Sensu users in our Community Slack, where you can ask questions and benefit from tips others picked up during their own Sensu Go migrations.
- [**Sensu Community Forum**][27]: Drop a question in our dedicated category for migrating to Go.
- [**Sensu Go Sandbox**][28]: Download the sandbox and try out some monitoring workflows with Sensu Go.
- [**Sensu translator**][23]: Use this command-line tool to generate Sensu Go configurations from your Sensu Core config files.

> We also offer **commercial support** and **professional services** packages to help with your Sensu Go migration. Learn more about [commercial support options][29] or contact our [Customer Reliability team][30].

Reviewer: There's no option to contact Reliability on the linked form, so let's suggest that they contact sales instead?

```suggestion
We also offer **commercial support** and **professional services** packages to help with your Sensu Go migration. Learn more about [commercial support options][29] or contact our [Sales team][30].
```

[18]: reference/filters
[19]: reference/checks/#check-hooks
[20]: /uchiwa/latest
[21]: https://blog.sensu.io/eol-schedule-for-sensu-core-and-enterprise
[22]: ../latest/migration/
[23]: https://github.com/sensu/sensu-translator

> [24]: https://docs.sensu.io/sensu-go

Author: ```suggestion
[24]: https://docs.sensu.io/sensu-go/latest/
```

> [25]: https://docs.sensu.io/sensu-go/5.14/installation/upgrade/#upgrading-to-sensu-go-from-sensu-core-1-x

Author: ```suggestion
[25]: https://docs.sensu.io/sensu-go/latest/installation/upgrade/#upgrading-to-sensu-go-from-sensu-core-1-x
```

[26]: https://slack.sensu.io/
[27]: https://discourse.sensu.io/c/sensu-go/migrating-to-go

> [28]: https://docs.sensu.io/sensu-go/5.14/getting-started/sandbox/

Author: ```suggestion
[28]: https://docs.sensu.io/sensu-go/latest/getting-started/sandbox/
```
